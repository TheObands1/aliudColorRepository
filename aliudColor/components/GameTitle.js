import React from "react";
import { Text, View, StyleSheet } from "react-native";

const GameTitle = () => {
    return(
        <View style={{ flexDirection: "row" }}>
            <Text style={[TitleStyle.fontStyle, { color: "#ecf0f1" }]}>aliud</Text>
            <Text style={[TitleStyle.fontStyle, { color: "#E64C3C" }, { textTransform: 'uppercase'}]}>c</Text>
            <Text style={[TitleStyle.fontStyle, { color: "#E57E31" }]}>o</Text>
            <Text style={[TitleStyle.fontStyle, { color: "#F1C431" }]}>l</Text>
            <Text style={[TitleStyle.fontStyle, { color: "#68CC73" }]}>o</Text>
            <Text style={[TitleStyle.fontStyle, { color: "#3998DB" }]}>r</Text>
        </View>
    );
}

const TitleStyle = StyleSheet.create({
    fontStyle: {
      fontSize: 50,
      fontFamily: "dogbyte"
    }
   });

export { GameTitle };