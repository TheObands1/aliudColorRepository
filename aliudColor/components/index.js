export * from './GameTitle'
export * from "./GameplayComponents/GameGrid"