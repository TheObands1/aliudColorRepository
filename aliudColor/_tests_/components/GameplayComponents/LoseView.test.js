import React from 'react';
import renderer from 'react-test-renderer';
import { LoseView } from '../../../components';
import {render, screen, act} from "@testing-library/react-native"
import { create } from 'react-test-renderer'

describe('tests of the Lose View', () => {
    beforeEach(() => {
        global.fetch = jest.fn(() => Promise.resolve({
            json: () => Promise.resolve ({
                data: "The Joke"
            })
        }))
    });
/*
    it("Loads the joke on mount", async () =>{
        await act(async () => create(<LoseView />));
        expect(screen.getByText("The Joke")).toBeInDocument();
    })
*/
    it('Render the component correctly ', () => {
        const tree = renderer.create(<LoseView />).toJSON();
        expect(tree).toMatchSnapshot();
      });

  });