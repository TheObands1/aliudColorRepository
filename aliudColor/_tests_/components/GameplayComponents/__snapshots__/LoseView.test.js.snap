// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests of the <LoseView/> Render the component correctly  1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "color": "#eee",
        "fontFamily": "dogbyte",
        "fontSize": 25,
        "marginTop": 20,
        "textAlign": "center",
        "textTransform": "capitalize",
      }
    }
  >
    you lost!, have a lame consolation joke
  </Text>
  <Text
    style={
      Object {
        "color": "#eee",
        "fontFamily": "dogbyte",
        "fontSize": 35,
        "marginTop": 20,
        "textAlign": "center",
      }
    }
  >
    
  </Text>
</View>
`;
